h2o-algos/src/main/java/hex/generic/GenericModelOutput.java

```diff
@@ -234,6 +241,58 @@ private static Object convertObjects(final Object source, final Object target) {
         return target;
     }
 
+    private static TwoDimTable convertModelParameters(final ModelParameter[] modelParameters) {
```

Comment:
I believe the MOJO part is done in a good way. However, couldn't we just reuse current API by filling _parms ? I know, the _parms is strongly typed and `GenericParameters` can't cover all the cases.

I'm thinking whether it would be possible to create static alternatives to all the param classes ? This way, a param once coded in could't be removed. Or provide two versions of `_parms` - a dynamic one like `_dynamic_parms` backed by a `TwoDimTable` (basically what you are doing here), but acting as a map. On frontend, just convert it. Or can we adjust the conversion on backend in order for the frontend client not to recognize anything ?